packages/relay/src/formatters.ts

```diff
@@ -113,6 +113,19 @@ const parseNumericEnvVar = (envVarName: string, fallbackConstantKey: string): nu
     return value;
 }
 
+/**
+ * Parse value from hex string to number, if value lenght is bigger than tinybar to weibar coef, it means that it's in weibar, which needs conversion to weibar.
+ * @param value 
+ * @returns parsedValue
+ */
+const valueHexToInt = (value: string): number => {
```

Comment:
Function name is misleading here. We should highlight the inner logic
```suggestion
const weibarHexToTinyBarInt = (value: string): number => {
```

Reply:
Good suggestion.